packages/udes-ui-core/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@udes-ui/core",
+  "version": "0.0.1",
+  "description": "UDES UI Core",
+  "author": "Richard Whitefoot <mouseholdonline@gmail.com>",
+  "license": "GPL-3.0-or-later",
+  "main": "dist/index.js",
+  "types": "lib/index.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unipartdigital/udes-ui.git"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "jest:lerna": "jest --coverage",
+    "test": "lerna run jest:lerna",
+    "lint": "eslint src -c .eslintrc.json --ext js,jsx,ts,tsx; exit 0",
+    "check": "lint-staged",
+    "build": "webpack --mode production"
+  },
+  "bugs": {
+    "url": "https://github.com/unipartdigital/udes-ui/issues"
+  },
+  "keywords": [
+    "udes",
+    "udes-ui"
+  ],
+  "dependencies": {
+    "@material-ui/core": "^4.7.0",
+    "@material-ui/styles": "^4.6.0",
+    "@types/react": "16.9.13",
+    "@types/styled-components": "^4.4.0",
+    "styled-components": "^4.4.1"
```

Comment:
Renovate requires us to pin npm packages to exact versions

Reply:
Thanks - fixed

Reply:
Thanks, fixed.